pythonforandroid/recipes/hostpython3/__init__.py

```diff
@@ -81,6 +82,15 @@ def build_arch(self, arch):
         build_dir = join(recipe_build_dir, self.build_subdir)
         ensure_dir(build_dir)
 
+        # ... And add that dir into the include flags
+        env = os.environ
+        cflags = " -I{build_dir}".format(build_dir=build_dir)
+
+        try:
+            env["CFLAGS"] += cflags
+        except KeyError:
+            env["CFLAGS"] = cflags
```

Comment:
:grimacing:, Shouldn't we put `includes` in `CPPFLAGS` instead of `CFLAGS`? teorically, the `macros` and the `includes` should go in `CPPFLAGS` right?

```suggestion
        cppflags = f" -I{build_dir}"

        try:
            env["CPPFLAGS"] += cppflags
        except KeyError:
            env["CPPFLAGS"] = cppflags
```

**Note:** Just noting that I changed to Python3's f-strings :wink:

Reply:
Ok. I lurked around the codebase and I saw that f-strings where not used. So I stuck with the style.

Reply:
Aaa..don't worry for strings formating, we have a kind of mixture right now..we have been supporting Python2 until a few month ago...

Reply:
I see 👌